docker/caffe2/jenkins/common/install_rocm.sh

```diff
@@ -42,6 +36,7 @@ install_ubuntu() {
                    hipblas \
                    rocrand \
                    rocm-profiler \
+                   hcsparse \
```

Comment:
what about hcrng?

Reply:
@bddppq Didn't see that PR. Yeah let's move all these changes there.

Reply:
Ah, I didn't see PR 9633.